src/qt/overviewpage.cpp

```diff
@@ -159,8 +159,8 @@ void OverviewPage::UpdateTip()
         tr("Make sure to keep your wallet updated."),
         tr("Backup your private key to recover your coins, using 'File' > 'Backup Wallet'"),
         tr("Always do your own research before using an external cryptocurrency service"),
-        tr("Never share your private key to an untrustworthy person."),
-        tr("Who owns the private keys owns the coins."),
+        tr("Never share your private key with anyone"),
+        tr("Who own the private keys own the coins."),
```

Comment:
Minor nit: I think the first `own` should remain `owns`. The change to the second looks correct.

Maybe "Whoever owns the private keys owns the coins." would be even better.